src/main/java/gregtech/api/mui/widget/GhostCircuitSlotWidget.java

```diff
@@ -135,8 +135,10 @@ private IPanelHandler getSelectorPanel() {
         return this.selectorPanel;
     }
 
-    private static class GhostCircuitSyncHandler extends ItemSlotSH {
+    private static class GhostCircuitSyncHandler extends PhantomItemSlotSH {
 
+        // TODO: should we be using this as it's marked internal? Or is it fine.
```

Comment:
Does anyone know why we are not just overriding `init` in PhantomItemSlotSH instead of using the internal marked method?